nightingale_utils/src/room_pose_lookup.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
```

Comment:
tip: consider running static analyzer on this script like pylint <pylint room_pose_lookup.py>. i find useful as it catches misspelled vars and just dumb mistakes generally